File: calc.dart
#!/usr/bin/env -S dart

import 'dart:core';
import 'dart:io';

void main() {
	loop: while (true) {
		final stack = [];

		for (final e in stdin.readLineSync()?.split(' ') ?? exit(0)) {
			final n = int.tryParse(e);
			if (n is int) {
				stack.add(n);
				continue;
			}
			
			if (stack.length < 2) {
				print("Too Few Arguments");
				continue loop;
			}

			final b = stack.removeLast();
			final a = stack.removeLast();

			switch (e) {
				case "+": stack.add(a + b);
				case "-": stack.add(a - b);
				case "*": stack.add(a * b);
				case "/": stack.add(a / b);
				default :
					print("Invalid Token");
					continue loop;
			}
		}

		if (stack.length == 1)
			print(stack.last);
		else
			print("Invalid Input");
	}
}

File: calc.py
#!/usr/bin/env python3

while True:
	stack = []

	try: s = input()
	except KeyboardInterrupt: exit()

	for e in s.split(" "):
		if e.isdigit() or e.startswith("-") and e[1:].isdigit():
			stack.append(int(e))
			continue

		if e in "+-*/":
			if len(stack) < 2:
				print("Too Few Arguments")
				break
			stack[-2:] = [int(eval(f"stack[-2] {e} stack[-1]"))]
		else:
			print("Invalid Token")
			break

	if len(stack) == 1:
		print(stack[0])
	else:
		print("Invalid Input")
